PubnativeLite/PubnativeLite/Core/Viewability/HyBidViewabilityManager.m

```diff
@@ -117,4 +118,9 @@ - (BOOL)isViewabilityMeasurementActivated {
     return OMIDPubnativenetSDK.sharedInstance.isActive && self.viewabilityMeasurementEnabled;
 }
 
+- (void)reportEvent:(NSString *)eventType {
+    HyBidReportingEvent* impressionOccurredEvent = [[HyBidReportingEvent alloc]initWith:eventType adFormat:nil properties:nil];
```

Comment:
Why adFormat and properties are set to nil here? Should not we get them as a parameter and set them here or?

Reply:
this reportEvent function is for the OMSDK events, which is not linked to adFormat. Properties are optionals. maybe in the future we need to add some other params. 
But for this case, the key event ( eventType) is enough for sending the OMSDK events ( same as in Android)